packages/pwa-kit-react-sdk/src/ssr/universal/components/storefront-preview/index.jsx

```diff
@@ -9,13 +9,15 @@ import React, {useEffect} from 'react'
 import PropTypes from 'prop-types'
 import {Helmet} from 'react-helmet'
 import {detectStorefrontPreview, getClientScript} from './utils'
-
+const defaultGetToken = () => {
+    return window.localStorage.getItem('token')
+}
```

Comment:
In my opinion, this is retail react app implementation detail that should not be leaked into the SDK... 🤔

Reply:
That's probably true, but given our goal of minimal customer effort for storefront preview and the fact that v2 is unlikely to see significant changes outside Storefront Preview support, I think it's a reasonable tradeoff.